### DISCUSSION TITLE
Version number in master branch is the same as last released version

### DISCUSSION THREAD

**Discussion by shaib** [encode/django-rest-framework#8567] [Potential Issue] [answered]
Hi,

What the subject says. This is an issue for me, because 
- I want to run tests against DRF as it is in master (among other reasons -- because I run against Django's main and #8539)
- I run my tests in dockers
- I have many dependencies, and in order to save both my time and PyPI bandwidth resources, I build my test image from a base image, which includes, among other things, the latest released version.
- When `pip`, after cloning the git repo, finds out that the version there is the same version it already has installed, it keeps the existing one. 

I suspect I'm not the only one in this position, but the current way version numbers are handled seems to have been going on for a long time, so I'm wondering if changing it may be considered.

**Comment by shaib** [comment, marked as answer]
Well, I can solve my issue by adding `--no-deps --ignore-installed` to the `pip install` command (`--ignore-installed` alone will happily also replace dependencies of DRF). But it feels a little dirty.

**Reply by lovelydinosaur** [reply]
> but the current way version numbers are handled seems to have been going on for a long time, so I'm wondering if changing it may be considered.

I guess we'd probably just want to leave things as they are yup.

Open to alternatives, but trying to keep changes to the project at a minimum really, so?... 🤷‍♂️
